### PR TITLE
add and use get_remote_url function

### PR DESCRIPTION
**Pull request by tcely**

### CODE REVIEW

**Comment by arusahni** [comment]
Hi! Thanks for submitting a PR. Could you please provide some context around what issue this is intended to address?

**Comment by tcely** [comment]
It is intended to let git resolve the final URL instead of misusing an intermediate value that git-req doesn't know how to handle anyway.

My remote.origin.url is gh:repo.git but asking git for the remote URL gets https://github.com/... as the code expects.

File: src/git.rs [hunk 1]
```diff
@@ -2,26 +2,29 @@ use std::path::Path;
...
+use git2::{Config, Error, Repository};
 use log::debug;
 use shellexpand;
 
 fn slugify_domain(domain: &str) -> String {
     str::replace(domain, ".", "|")
 }
 
+pub fn get_remote_url(remote: &str) -> String {
+    cmd!("git", "remote", "get-url", remote).read().unwrap()
```
[the remark this answers is not in the document]
Comment (tcely) [reply src/git.rs]: If that uses the libgit2 git_remote_url function then it should work.

**Comment by arusahni** [comment]
Can you grant me push access to your PR? https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork I'd like to add my commit so you can verify it works for you.

**Comment by tcely** [comment]
That feature was already enabled.

**Comment by arusahni** [comment]
Sorry - the mistake was on my end. I missed the `-1` suffix on the branch name. I've pushed a commit to it. If it works for you, I'll merge this PR.

**Comment by tcely** [comment]
It works for me.

**Comment by arusahni** [comment]
Thanks for your contribution, @tcely! I hope to have a release cut by the end of this coming week.

**Comment by arusahni** [comment]
@tcely sorry for the delay - I had computer issues. The remote URL bugfix is in the newly-released v2.0.1. Thanks for the PR!

**Pull request merged**